lib/smart_answer/calculators/marriage_abroad_data_query.rb

```diff
@@ -12,6 +12,10 @@ class MarriageAbroadDataQuery
       france monaco new-caledonia wallis-and-futuna
     )
 
+    FRENCH_OVERSEAS_TERRITORIES_OFFERING_PACS = %w(
+      new-caledonia wallis-and-futuna
+    )
+
```

Comment:
This was because I was trying to avoid the long lines that the existing style leads to. However, consistency is more important so I've changed these in https://github.com/alphagov/smart-answers/pull/2470/commits/1a316a1b8d8f6b4c2b8733750c588559937263bc.